### PR TITLE
issue634/bug-timeouts-in-gbs-pool-tube-creation/low_risk

### PR DESCRIPTION
**Pull request by JamesGlover**
- Nginx is attempting to render pages for 502 and 504, but was failing
because they were missing. This resulted in a confusing 404.
- Added some guidelines on how best to inform us about errors.

This coveres some low-risk independently deployable improvements spun out of 634. Actual code changes to address the performance issues to follow.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,9 +58,9 @@ There are a few tools available to assist with writing specs:
 * Strategies: You can use json `:factory_name` to generate the json that the API is expected to receive. This is very useful for mocking web responses. The association strategy is used for building nested json, it will usually only be used as part of other factories.
 
 * Traits:
-    * `api_object`: Ensures that lots o the shared behaviour, like actions and uuids are generated automatically
+  * `api_object`: Ensures that lots o the shared behaviour, like actions and uuids are generated automatically
 barcoded: Automatically ensures that barcode is populated with the correct hash, and calculates human and machine barcodes
-    * `build`: Returns an actual object, as though already found via the api. Useful for unti tests
+  * `build`: Returns an actual object, as though already found via the api. Useful for unti tests
```
Comment (andrewsparkes) [inline README.md:63]: typo unti -> unit

File: README.md [hunk 2]
```diff
@@ -64,9 +58,9 @@ There are a few tools available to assist with writing specs:
 * Strategies: You can use json `:factory_name` to generate the json that the API is expected to receive. This is very useful for mocking web responses. The association strategy is used for building nested json, it will usually only be used as part of other factories.
 
 * Traits:
-    * `api_object`: Ensures that lots o the shared behaviour, like actions and uuids are generated automatically
+  * `api_object`: Ensures that lots o the shared behaviour, like actions and uuids are generated automatically
```
Comment (andrewsparkes) [inline README.md:61]: typo lots o -> lots of

File: public/502.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+      .status-code {
+        color: rgb(217, 83, 79);
+      }
+
+    </style>
+  </head>
+
+  <body>
+    <!-- This file lives in public/500.html -->
```
Comment (andrewsparkes) [inline public/502.html:39]: Comment needed/correct? 502?

**Review by andrewsparkes — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (JamesGlover) [reply README.md]: Fixed

File: public/502.html (hunk 3 above)
Comment (JamesGlover) [reply public/502.html]: Yep, fixed.

File: README.md (hunk 2 above)
Comment (JamesGlover) [reply README.md]: Fixed

**Pull request merged**